app/src/main/res/layouts/onboarding/layout/onboard3_location.xml

```diff
@@ -3,128 +3,177 @@
-                android:background="@color/common_google_signin_btn_text_dark_disabled" />
 
-        <TextView
-            android:id="@+id/tv3"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="10dp"
-            android:text="Set Work Location"
-            android:textAlignment="center"
-            android:textAppearance="@style/TextAppearance.AppCompat.Body2"
-            android:textSize="18sp" />
+        </RelativeLayout>
 
         <RelativeLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="0dp"
+            android:layout_marginTop="16dp"
+            android:background="@android:color/white"
+            android:layout_weight="1"
+            >
 
             <com.google.android.gms.maps.MapView
+                xmlns:map="http://schemas.android.com/apk/res-auto"
                 android:id="@+id/mapWork"
                 android:layout_width="match_parent"
-                android:layout_height="160dp"
-                android:layout_marginEnd="15dp"
-                android:layout_marginStart="15dp" />
+                android:layout_height="match_parent"
+                android:visibility="invisible"
+                android:layout_marginBottom="85dp"
+                map:cameraZoom="12"
+                map:mapType="normal"
+                map:liteMode="true" />
+
+            <TextView
+                android:id="@+id/warning_location_work"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_alignParentTop="true"
+                android:layout_marginTop="30dp"
+                android:text="@string/onboarding_no_location"
+                android:textAlignment="center"
+                android:textAppearance="@android:style/TextAppearance.Material.Medium"
+                android:textStyle="italic" />
+
+            <TextView
+                android:id="@+id/work_location_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/mapWork"
+                android:layout_marginStart="10dp"
+                android:layout_marginTop="-80dp"
+                android:text="@string/onboarding_location_work_label"
+                android:textAppearance="@android:style/TextAppearance.Material.Medium"
+                android:textColor="@android:color/black" />
+
+            <TextView
+                android:id="@+id/work_location_address"
+                android:layout_width="180dp"
+                android:layout_height="match_parent"
+                android:layout_alignStart="@+id/work_location_label"
+                android:layout_below="@+id/work_location_label"
+                android:text="@string/onboarding_location_default_address"
```

Comment:
Remove this, it is not needed since we have it in the `MapView`

Reply:
I disagree. I think that if something is going to show up in that spot it's good UX to have a zero-state before the user selects anything.
My suggestion is for @jlsilva1994 to make to the text like `No location selected.` instead of the 3 dots and non-capital start.